### PR TITLE
update functionality and page done

### PR DESCRIPTION
**Pull request by mikeyliang**

### CODE REVIEW

File: src/routes/project/create/+page.svelte [hunk 1]
```diff
@@ -65,8 +65,11 @@
 		selectedMembers = currentMember?.map((member) => ({
 			value: member.id,
 			label: `${member.firstName} ${member.lastName}`,
-			canRemove: true
+			canRemove: false
 		}));
+
+		console.log(selectedMembers);
+		console.log(members);
```
Comment (jay-joo-code) [inline src/routes/project/create/+page.svelte:72]: let's remove console logs

**Review by jay-joo-code — CHANGES_REQUESTED** [review]
overall looks good!
just need to remove console logs

File: src/routes/project/create/+page.svelte (hunk 1 above)
Comment (mikeyliang) [reply src/routes/project/create/+page.svelte:72]: okay!! 👍

**Review by jay-joo-code — DISMISSED** [review]
looks good!

**Comment by jay-joo-code** [comment]
@mikeyliang looks good!
just checking, currently how would a user edit their project? I'm not sure how they would be able to navigate to `/project/update/:projectID` path

**Comment by mikeyliang** [comment]
> @mikeyliang looks good! just checking, currently how would a user edit their project? I'm not sure how they would be able to navigate to `/project/update/:projectID` path

I think in the menu they would be able to go to a profile page that has all their projects? Then each one of these projects links to a particular projectId, then redirect to /project/update/:projectID?

**Comment by jay-joo-code** [comment]
> I think in the menu they would be able to go to a profile page that has all their projects? Then each one of these projects links to a particular projectId, then redirect to /project/update/:projectID? lmk what you think

Sounds good! Feel free to build what you think makes most sense. I just wanted to check in with you that that's not within the scope of this PR. 
This PR is already pretty big. Good work!

**Pull request merged**